spark_expectations/utils/reader.py

```diff
@@ -168,15 +169,24 @@ def get_rules_from_df(
             self._context.reset_num_row_dq_rules()
             self._context.reset_num_query_dq_rules()
 
+            if not params:
+                params = {}
+            else:
+                for key, value in params.items():
+                    rules_df = rules_df.withColumn(
+                        "product_id",
+                        regexp_replace("product_id", "\\{" + key + "\\}", value),
```

Comment:
Can we please add information to the documentation about which columns can have parameterization when we create rules!

Reply:
updated comments in documentation